app/tests/system/assertions/global.js

```diff
@@ -0,0 +1,53 @@
+/*
+ Global assertions for any webview context.
+*/
+
+var GlobalAssertions = function(browser) {
+    this.browser = browser;
+};
+
+/*
+ Assert that landscape mode is disabled.
+*/
+GlobalAssertions.prototype.assertPortrait = function() {
+    var self = this;
+    this.browser
+        .contexts(function(result) {
+            self.browser
+                .setContext(result.value[result.value.length - 1])
+                .setOrientation('LANDSCAPE')
```

Comment:
Nope. We set the orientation to LANDSCAPE and check that the value of `window.orientation` is unchanged.